tests/by-util/test_dd.rs

```diff
@@ -1139,3 +1139,48 @@ fn test_block_sync() {
         .stdout_is("012  abcde     ")
         .stderr_is("2+1 records in\n0+1 records out\n1 truncated record\n");
 }
+
+#[test]
+fn test_bytes_count_bytes_iflag() {
+    new_ucmd!()
+        .args(&["conv=swab", "count=14", "iflag=count_bytes"])
+        .pipe_in("0123456789abcdefghijklm")
+        .succeeds()
+        .stdout_is("1032547698badc");
+}
+
+#[test]
+fn test_bytes_skip_bytes_iflag() {
+    new_ucmd!()
+        .args(&["skip=10", "iflag=skip_bytes"])
+        .pipe_in("0123456789abcdefghijklm")
+        .succeeds()
+        .stdout_is("abcdefghijklm");
+}
+
+#[test]
+fn test_bytes_skip_bytes_pipe_iflag() {
+    new_ucmd!()
+        .args(&["skip=10", "iflag=skip_bytes", "bs=2"])
+        .pipe_in("0123456789abcdefghijklm")
+        .succeeds()
+        .stdout_is("abcdefghijklm");
+}
+
+#[test]
+fn test_bytes_oseek_bytes_oflag() {
+    new_ucmd!()
+        .args(&["seek=8", "oflag=seek_bytes", "bs=2"])
+        .pipe_in("abcdefghijklm")
+        .succeeds()
+        .stdout_is_fixture_bytes("dd-bytes-alphabet-null.spec");
+}
+
+#[test]
+fn test_bytes_oseek_bytes_trunc_oflag() {
+    new_ucmd!()
+        .args(&["seek=8", "oflag=seek_bytes", "bs=2", "count=0"])
+        .pipe_in("abcdefghijklm")
+        .succeeds()
+        .stdout_is_fixture_bytes("dd-bytes-null-trunc.spec");
+}
```

Comment:
missing tests for `iseek`, and missing tests for the additive behavior of `oseek` and `iseek` that appears in the code

Reply:
Looks like I mixed up tests while checking in. Adding the iseek tests, and removing the three unneeded tests.
Good point on wanting to test the additive flags. Writing tests for that.